src/components/Comment/Comment.tsx

```diff
@@ -14,24 +14,23 @@ type Props = {
   pillar: Pillar;
 };
 
-const commentControls = (pillar: Pillar) => css`
+const commentControls = css`
   list-style: none;
   ${textSans.xsmall()};
+  display: flex;
+  align-items: flex-start;
+`;
 
-  * {
-    display: inline-block;
-  }
-
-  *:not(:last-child) {
-    font-weight: bold;
-    margin-right: ${space[2]}px;
-    color: ${palette[pillar][400]};
-  }
+const commentControlsButton = (pillar: Pillar) => css`
+  font-weight: bold;
+  margin-right: ${space[2]}px;
+  color: ${palette[pillar][400]};
+  border: 0;
+`;
 
-  *:last-child {
-    float: right;
-    font-weight: normal;
-  }
+const controleWrapper = css`
```

Comment:
Nitpick: it's 'control', not 'controle'. But maybe 'spaceBetween' is more descriptive?